README.md

```diff
@@ -23,7 +23,7 @@ Fork from https://github.com/urcomputeringpal/actions-docker
 - Create a Secret on your repository named `GCLOUD_SERVICE_ACCOUNT_KEY` (Settings > Secrets) with the contents of:
 
 ```shell
-echo -n "$(cat path-to/downloaded-key/4a276e9e5862.json)" | base64
+cat path-to/key.json | base64 -w 0
```

Comment:
just out of curiosity, why do you use this option?

Reply:
It will provide you the encoded base64 in one line instead. This will avoid having a broken key.json when reading it inside the Docker image. When you decode your base64, it produces a new line `\n`